src/data/aboutData.tsx

```diff
@@ -0,0 +1,638 @@
+import {
+  Text,
+  Link,
+  List,
+  Heading,
+  Accordion,
+  Box,
+} from "@nypl/design-system-react-components";
+import JWPlayer from "@jwplayer/jwplayer-react";
+
+const aboutData = [
+  {
+    heading: (
+      <Heading
+        sx={{ marginTop: "xxl" }}
+        level="h1"
+        size="heading1"
+        text="About NYPL Digital Collections"
+      />
+    ),
+    body: (
+      <Text sx={{ marginBottom: "xxl", marginTop: "s" }}>
+        {" "}
+        The New York Public Library (NYPL) Digital Collections platform is the
+        primary portal for engaging with our digitized collections and their
+        descriptions, over 922,495 items and counting. While that is a small
+        fraction of the Library&apos;s overall holdings, it is representative of
+        the diversity of our vast collections—from books to videos, maps to
+        manuscripts, illustrations to photos, and more.
+      </Text>
+    ),
+  },
+  {
+    heading: <Heading level="h2" size="heading4" text="Items & permissions" />,
+    body: (
+      <Text sx={{ marginBottom: "xxl", marginTop: "s" }}>
+        {" "}
+        We actively review and label materials in Digital Collections with
+        statements that indicate how you may reuse items, and what sort of
+        permission, if any, you need to do so.
+      </Text>
+    ),
+  },
+  {
+    heading: <Heading level="h2" size="heading4" text="Download items" />,
+    body: (
+      <Box sx={{ marginBottom: "xxl" }}>
+        <Text sx={{ marginTop: "s" }}>
+          To download, navigate to the Download Options section under each item.
+          Simply click on your preferred file size and check your browser&apos;s
+          download folder for the image.{" "}
+        </Text>
+        <JWPlayer
+          playerId="KRF3T2Kr"
+          library="https://cdn.jwplayer.com/libraries/KRF3T2Kr.js"
+          file={
+            "https://digitalcollections-prod.s3.amazonaws.com/assets/pd_download-f4513d03a49cd6a20013cd7a6b73e8a1.mp4"
+          }
+        />
+      </Box>
+    ),
+  },
+  {
+    heading: <Heading level="h2" size="heading4" text="Find items" />,
+    body: (
+      <Box sx={{ marginBottom: "xxl" }}>
+        <Text sx={{ marginTop: "s" }}>
+          {" "}
+          Looking for something? Start with a{" "}
+          <Link href="https://digitalcollections.nypl.org">search</Link> or
```

Comment:
Hmm eventually these should be self-referential and be `"/"`, `"/search/index..."`, etc. You could use the `DC_URL` value from appConfig but those urls are also not self-referential. Can you add a TODO comment somewhere so we don't forget? This won't have to be done until this app is entirely complete.